content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,255 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+In machine learning projects, number of <abbr>experiments</abbr> grow rapidly.
+DVC can track these experiments, list their most relevant parameters and
+metrics, and commit only the ones that we need to Git.
+
+In this section, we explore the basic features of DVC experiment management with
+[`get-started-experiments`][gse] project.
+
+[gse]: https://github.com/iterative/get-started-experiments
+
+<details>
+
+## ⚙️ Installing the Example Project
+
+These commands are run in the [`get-started-experiments`][gse] project. You can
+run the commands in this document after cloning the repository and installing
+the requirements.
+
+### ⚡ Clone the project and create virtual environment
+
+Please clone the project and create a virtual environment.
+
+> We create a virtual environment to keep the libraries we use isolated from the
+> rest of your system. This prevents version conflicts.
+
+```dvc
+$ git clone https://github.com/iterative/get-started-experiments -b get-started
```

Comment:
why do we need `-b`?

Reply:
idk - branch `get-started` doesn't even exist at https://github.com/iterative/example-dvc-experiments/branches

Reply:
It's a tag. https://github.com/iterative/example-dvc-experiments/tree/get-started

It shows the point to start this tutorial. I'm moving the tag to a particular commit instead of updating the name here.

Reply:
okay, question - since we are not creating this repo from the very beginning like we do in the example get started one - does it make sense to let them just clone the latest version (and may be briefly explain core pieces of it)?

Reply:
We should start somewhere in the middle, before experiments. Otherwise, the experiments repo won't contain any experiments. We are also using this repository for `dvc exp pull/push` and I believe it's better to keep this tag and start from it. The alternative, that is starting from a repo with experiments in it leads to a more complicated narrative.

Reply:
okay, makes sense ... I wonder if long term we can make this tag `get-started` the default state though fo the repo (and add experiments as "you go" in the docs)